code/modules/clothing/clothing.dm

```diff
@@ -471,6 +474,9 @@ BLIND     // can't see anything
 						flavour = "[adjust_flavour]"
 					to_chat(user, "You [flavour] \the [src].")
 					suit_adjusted = 1 //Suit's adjusted.
+					for(var/X in actions)
+						var/datum/action/A = X
+						A.UpdateButtonIcon()

```

Comment:
You're probably wondering why I update the action buttons in two separate instances instead of down the tree.